src/sidebar/services/groups.js

```diff
@@ -348,25 +323,103 @@ export default function groups(
       directLinkedGroupId
     );
 
+    addGroupsToStore(groups, [
+      directLinkedAnnotationGroupId,
+      directLinkedGroupId,
+      store.getDefault('focusedGroup'),
+    ]);
+
+    return groups;
+  }
+
+  /**
+   * Load the specific groups configured by the annotation service.
+   *
+   * @param {string[]} groupIds
+   */
+  async function loadServiceSpecifiedGroups(groupIds) {
+    let error;
+    const tryFetchGroup = async id => {
+      try {
+        return await fetchGroup(id);
+      } catch (e) {
+        error = e;
```

Comment:
As implemented, this error would supersede any error from an earlier group fetch. Is that as intended?

Reply:
Yes. I think that showing multiple notifications is likely to be too noisy, so we just capture one. I've arbitrarily chosen to keep the last one.